src/main/java/calculator/Message.java

```diff
@@ -0,0 +1,32 @@
+package calculator;
+
+import java.util.HashMap;
+
+/**
+ * Message class
+ * - message output class
+ *
+ * return type : void
+ */
+public class Message {
+    HashMap<String, String> exceptionMessageList = new HashMap<String, String>();
```

Comment:
객체지향의 유명한 규칙중 `Program to Interface` 라는 규칙이 있습니다.

```suggestion
    Map<String, String> exceptionMessageList = new HashMap<>();
```

구체클래스가 아닌 인터페이스로 사용하는것은 어떤 장점이 있을까요?